<launch>
  <node name="sonar_sensor" pkg="four_wheel_drive" type="publisher" output="screen"/>
  <node name="dummy_commander" pkg="four_wheel_drive" type="publisher" output="screen"/>
  <node name="motor_driver" pkg="four_wheel_drive" type="subscriber" output="screen"/>
</launch>